database/nakadi/10_data/04_tables/01_event_type.sql

```diff
@@ -1,7 +1,10 @@
 CREATE TABLE IF NOT EXISTS zn_data.event_type (
   et_name varchar(255) NOT NULL PRIMARY KEY CHECK (et_name <> ''),
+  et_topic varchar(255) NOT NULL,
```

Comment:
If you take it out from EventType POJO - you will loose caching and will have to take topic name from DB each time :)